pydicer/convert/headers.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import json
+import pydicom
+
+logger = logging.getLogger(__name__)
+
+
+def convert_dicom_headers(dcm_file, binary_path, json_file):
+    """Save the DICOM Headers as a JSON file
+
+    Args:
+        dcm_file (str|pathlib.Path): The files from which to save the headers.
+        binary_path (str): Relative path to binary data which will be placed into JSON.
+        json_file (str|pathlib.Path): Path to JSON file to save output.
+    """
+
+    # Write the DICOM headers (of the first slice) to JSON
+    dcm_ds = pydicom.read_file(dcm_file)
+    dcm_dict = dcm_ds.to_json_dict(
```

Comment:
This class is great, nice and tidy! 
For image the export looks concise and tidy. 
For Plan and structure it produces a very large file with many big floating point arrays for some keys, maybe we can simply those exports to a limited list of attributes?

Reply:
Yep definitely agree for the structure sets. But it won't be as simple as remove certain top level attributes. Because we want to keep information about structures we haven't converted (such as points) but remove the contours for which we now have the nifiti masks.

For plans I would say we shoudl keep everything. Since we don't convert anything to nifti, we want to make sure that all the plan data is there to call upon if needed.

But we can discuss how this might work in our next catch up.